backend-shared/divesummary.cpp

```diff
@@ -9,9 +9,9 @@ QStringList diveSummary::diveSummaryText;
 
 timestamp_t diveSummary::firstDive, diveSummary::lastDive;
 int diveSummary::dives[2], diveSummary::divesEAN[2], diveSummary::divesDeep[2], diveSummary::diveplans[2];
-long diveSummary::divetime[2], diveSummary::depth[2], diveSummary::sac[2];
-long diveSummary::divetimeMax[2], diveSummary::depthMax[2], diveSummary::sacMin[2];
-int diveSummary::divesSAC[2];
+long diveSummary::divetime[2], diveSummary::depth[2];
+long diveSummary::divetimeMax[2], diveSummary::depthMax[2], diveSummary::sacMin[2], diveSummary::sacMax[2];
```

Comment:
sacMax is really not needed. and if needed how come minDepth is not needed ?

Reply:
I guess you and I just differ in our opinion on what is useful information.
The shallowest dive I have done? I wouldn't know who cares about this.
The highest SAC rate I had on a dive? Yeah, when planning dives it is extremely useful to have an upper boundary for the SAC rate to make sure you don't run out of gas. I think that value is much more useful in real life than the minimum SAC rate (which is basically a vanity number - yay, my min SAC is 7l/min. Great. So what?)